README.md

```diff
@@ -113,6 +112,7 @@ See instructions [here](https://github.com/NVIDIA/nvidia-docker).
 
 
 ### Running Docker Container (Interactive)
+To use the Docker container as an interactive virtual environment, you can run a container mounted to your local data and code directories and execute the `bash` program. This may be useful if you are in the process of developing a submission.
```

Comment:
would it make more sense to rewrite this as:

"To use the Docker container as an interactive virtual environment, you can run a container with your local data and code directories mounted and execute the `bash` program. This may be useful if you are in the process of developing a submission."

Reply:
yes that is easier to read. thanks!